tests/UI/pages/BO/catalog/attributes/index.js

```diff
@@ -25,7 +25,7 @@ class Attributes extends BOBasePage {
 
     // Header selectors
     this.addNewAttributeLink = '#page-header-desc-attribute_group-new_attribute_group';
-    this.addNewValueLink = '#page-header-desc-attribute_group-new_value';
+    this.addNewValueLink = '// a[data-role="page-header-desc-attribute_group"]';
```

Comment:
```suggestion
    this.addNewValueLink = 'a[data-role="page-header-desc-attribute_group"]';
```

I think it was xpath that you used

Reply:
```suggestion
    this.addNewValueLink = '[data-role=page-header-desc-attribute_group]';
```

I think you can even delete the tag